### DISCUSSION TITLE
New to telemetry radios....what do the abbreviations mean?

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4561] [Q&A] [answered]
Hey everyone! I recently purchased a RadioMaster MT12 and after setting up some telemetry I see many abbreviations I don't know and can't find on the web. 

Obviously RSSI is the signal strength and BATT is battery level, but what about ones like "**TRSS**", "**RQly**", "**RNse**" "**RSNR**"?

Help is much appreciated!

**Comment by pfeerick** [comment, marked as answer]
The sensors available are RF protocol specific... ELRS? 

- T(ransmitter) RSS(I)
- R(eceiver Link) Q(ua)l(it)y - percentage that is meant to be a bit more human friendly than RSSI/RNse/RSNR
- R(eceiver) N(oi)se
- R(eceiver) S(ignal to) N(oise) R(atio)

**Reply by inventor7777** [reply]
Thank you very much! That answers my question.

**Comment by BlackBird11** [comment]
what does ant, rfmd, tsnr stand for?

**Reply by pfeerick** [reply]
Like with the first question, these are protocol specific... so you really need to say what system you are using when asking questions like that. ;) However, assuming this is ELRS:

- Ant(enna) - to indicate which RX antenna is active if using a dual/diversity antenna RX
- RFMD (RF Mode) - relates the package rate you have configured => https://www.expresslrs.org/info/signal-health/#rf-mode-indexes-rfmd
- T(ransmitter) S(ignal to )N(noise )R(ratio) - how well the transmitter can haer the receiver

The full list is here...
https://www.expresslrs.org/faq/?h=tsnr#what-does-rqly-tqly-rssi-x2-snr-x2-mean
